src/Pages/ReferenceTokens.razor

```diff
@@ -97,4 +118,28 @@
         auditLog.Log("ReferenceTokenChecked", data);
         _referenceTokenData = referenceTokenValidator.Validate(_checkTokenModel);
     }
+
+    private void OnRevokeDialogClose(bool accepted)
+    {
+        if (accepted)
+        {
+            Dictionary<string, string> data = new()
+            {
+                { "ClientId", _clientIdToRevoke },
+                { "ReferenceTokenId", _referenceTokenIdToRevoke.ToString() }
+            };
+        
+            auditLog.Log("ReferenceTokenRevoke", data);
```

Comment:
Perhaps the auditing should be part of the `IReferenceTokenRevoker` implementation as well. Maybe we would want to revoke a reference token from somewhere else in the UI as well and then we'd have to duplicate the auditing logic.

Reply:
fixed it in 23ec94e, also did the same with the IReferenceTokenValidator